packages/experimental/Switch/src/SwitchTokens.android.ts

```diff
@@ -0,0 +1,61 @@
+import { Theme } from '@fluentui-react-native/framework';
+import { TokenSettings } from '@fluentui-react-native/use-styling';
+import { SwitchTokens } from './Switch.types';
+
+export const defaultSwitchTokens: TokenSettings<SwitchTokens, Theme> = (t: Theme) => ({
+  borderRadius: 100,
+  thumbSize: 26,
+  thumbRadius: 100,
+  trackHeight: 32,
+  trackWidth: 52,
+  thumbMargin: 3,
+  fontSize: t.typography.variants.body1.size,
+  fontWeight: t.typography.variants.body1.weight,
+  fontLineHeight: t.typography.variants.body1.lineHeight,
+  color: t.colors.neutralForeground1,
+  elevation: t.shadows.shadow8.key.blur,
```

Comment:
"Can this take a shadow alias token from fluent? If not we should see if we can change things to be able to do that, for example by using the Shadow component." - Comment from other review comment 

@rurikoaraki , I have use the shadow from the theme here , and it is access in .styling file. 
you mean this taking shadow from fluent token ? 

On the android the shadow component that is made for iOS doesn't work for android.  

The way iOS shadow work is they depend upon few properties as : 
shadowColor: "#000",
shadowOffset: {
	width: 0,
	height: 2,
},
shadowOpacity: 0.25,
shadowRadius: 3.84,

which is mapped on the shadow component and works but on the android the shadow is created using only one property "elevation" and upon discussion with Fluent android team , they said there still in process for mapping of these values to the elevation value but taking shadow like will work

Reply:
As related to design from android we're following elevation value application based on shadow.key.blue value.